app/components/Form/Functions/getMilestoneFilteredSchema.ts

```diff
@@ -35,9 +35,14 @@ export const getMilestoneFilteredSchema = (
   const newDataObject: any = {};
 
   for (const key of Object.keys(filteredSchema.properties)) {
+    const [updatedFormData, prevFormData] = [
```

Comment:
Nice, two bug fixes in one! No suggested change here, but FYI, there's a similar thing going on in the `schemaFilteringHelpers`